src/main/java/org/folio/oaipmh/helpers/InventoryStorageHelper.java

```diff
@@ -79,4 +79,9 @@ public String buildItemsEndpoint(Request request) throws UnsupportedEncodingExce
 
     return "/instance-storage/instances" + queryBuilder.build() + "&limit=" + RECORDS_LIMIT;
   }
+
+  @Override
+  public String getInstanceEndpoint(String id) {
+    return "/instance-storage/instances?query=source==MARC+and+id=" + id;
```

Comment:
Due to the fact that we are interested only in marc fromat we have to specify this info to instance storage service. The `/instance-storage/instances/:id` endpoint does not allow to specify any query. So there were 3 ways to resolve this:
- Use `/instance-storage/instances` with query parameter (implemented approach)
- Use `/instance-storage/instances/:id` and check if the returned instance has "source" == "MARC"
- Use `/instance-storage/instances/:id/source-record/marc-json` to get marc record

In case the chosen approach is not efficient enough, please let me know.

Reply:
The chosen approach seems to be the most efficient one.